apps/condo/domains/acquiring/schema/AcquiringIntegration.js

```diff
@@ -86,6 +86,18 @@ const AcquiringIntegration = new GQLListSchema('AcquiringIntegration', {
             options: CONTEXT_STATUSES,
             defaultValue: CONTEXT_IN_PROGRESS_STATUS,
         },
+        vatPercentOptions: {
+            schemaDoc: 'Comma separated values of VAT. Set by system administrators.',
+            adminDoc: 'For example: "0,10,20" is for RU',
+            type: 'Text',
+            isRequired: false,
+            hooks: {
+                resolveInput: async ({ resolvedData, fieldPath }) => {
```

Comment:
`resolveInput` is triggered each time models changed (create/update). So all changed integrations will have `''` as a default value because of that. But it should probably be `null`, since it's not required. 

Even considering that the empty string is a valid argument, autocomplete with it makes the api inconsistent

Reply:
fixed